src/app/modules/angular-slickgrid/services/bsDropdown.service.ts

```diff
@@ -0,0 +1,85 @@
+
+import { Injectable } from '@angular/core';
+import { AngularGridInstance } from './../models/index';
+import { AngularUtilService } from './angularUtil.service';
+
+
+interface DropDownServiceParams {
+  grid: AngularGridInstance,
```

Comment:
You don't need to pass this `grid` object, you can get it instead from the `SharedService`, see this [line](https://github.com/ghiscoding/Angular-Slickgrid/blob/master/src/app/modules/angular-slickgrid/services/extension.service.ts#L48) as example

```ts
constructor(... sharedService: SharedService) {}
...
this.sharedService.grid;
```